pipeline/special_sauce.rules

```diff
@@ -37,11 +37,11 @@ if "mhc_alleles" in config["input"] and _rna_exists():
       input:
         vcfs = _get_vaxrank_input_vcfs,
         rna = join(WORKDIR, "rna.bam"),
-        rna_index = join(WORKDIR, "rna.bam.bai")
```

Comment:
Removed the comma. I think the whitespace change might be worth leaving since there was an odd indentation occurring for that comment (possibly a tab rather than spaces)